### ISSUE TITLE
Termine / Veranstaltungen

### ISSUE THREAD

**Issue by pascalwittler** [entropia/gewerbehof-webseite#22]
Auf der Startseite könnten anstehende Termine und Veranstaltungen dargestellt werden. Auf jedem Plenum, auf Anfrage per Mail oder per Pull Request könnten dann Betriebe und Vereine des Gewerbehofs ihre nächsten Termine auf der gemeinsamen Website veröffentlichen.

Das würde den Eindruck eines lebendigen, aktiven Vereins (Gewerbehof = Verein) nach außen sicher befördern.

Denkbare Termine für die Liste wären aus meiner Sicht:
* Gemeinsame Veranstaltungen im Gewerbehof (z.B. Gewerbehof-/Sommerfest)
* Veranstaltungen einzelner Betriebe und Vereine (z.B. Entropia Hackerfryhstyck)
* denkbar: "interne" Termine als Reminder, z.B. Gewerbehofplenum (ob gewollt kann ich schwer beurteilen)
* denkbar: Urlaub/Ferien von Betrieben (Werbewirksamkeit vs. Informationsangebot bedenken)

**Comment by orat** [comment]
Gute Idee, die als "denkbar" beschrifteten Punkte würde ich aber nicht aufnehmen: intern sollte intern bleiben und Ferien/Urlaub von Betrieben gehört meiner Meinung nach auf die Webseiten der jeweiligen Betriebe.

**Comment by soc** [comment]
Hab mal das Gewerbehoffest nach Aktivitäten gepackt, gut so?

**Comment by orat** [comment]
Schön wäre es wenn wir einen gut strukturierten und leicht aktualisierbaren Veranstaltungskalender hätten. Vielleicht lohnt es sich, sich auf die Suche nach passendem Code/Plugins zu machen.  Irgendwas tabellarisches wo man draufklicken kann uns sich dann eine Seite mit Details öffnet. Ausserdem wäre es gut wenn man mit einem Klick einen kompletten Veranstaltungskalidern abbonieren könnten. Wir haben ja jetzt auch die GWH-owncloud. Da könnten wir ja einen Kalender pflegen und es ermöglichen diesen Kalender über die Webseite zu abbonieren.

**Comment by orat** [comment]
GWH-Fest in Aktivitäten ist erst mal gut so, zumindest als Platzhalter wenn die Webseite mal auf dem Plenum besprochen wird.

**Comment by pascalwittler** [comment]
Mittelfristig ist so etwas auch für die (im Wesentlichen statische) landing page für Entropia in Diskussion. Bis die Strukturen (Kalender in in der GH-nextcloud, nennenswerte Anzahl von Veranstaltungen) geschaffen sind, habe ich hier bestimmt einen Stand, von dem man so eine Funktion für die GH-Website ableiten kann. Bis dahin finde ich eine händisch gepflegte Seite nicht verkehrt.

**Comment by orat** [comment]
Ja, das klingt doch perfekt so. Eine zeitlang händisch pflegen mit Perspektive einen "richtigen" Kalender einzubinden ist genau richtig. Hauptsache wir blockieren durch das Warten auf einen Kalender oder irgendeine andere Funktionalität nicht den Wechsel auf die neue Seite. Ich hab mal nach Code für mkdocs gesucht mit Kalenderfunktionalität, bin da aber nicht fündig geworden. Überhaupt scheint es mir da nicht sonderlich viele Erweiterungen zu geben. Die Python-API um eigene Erweiterungen zu schreiben schaue ich mir bei Gelegenheit mal an.

**Comment by soc** [comment]
Ich denke das sinnvollste ist wohl ein Skript zu suchen, das iCal liest, und diesem dann die iCal-Addresse vom Kalender der GH-Cloud zu geben, das muss dann nicht MkDocs-spezifisch sein. MkDocs konzentriert sich ja eher auf die statische Seitengenerierung.

**Comment by pascalwittler** [comment]
> Ich denke das sinnvollste ist wohl ein Skript zu suchen, das iCal liest, und diesem dann die iCal-Addresse vom Kalender der GH-Cloud zu geben, das muss dann nicht MkDocs-spezifisch sein. MkDocs konzentriert sich ja eher auf die statische Seitengenerierung.

Das ist genau, was ich für die zukünftige Entropia landing page auch vorgeschlagen habe.

**Comment by orat** [comment]
Gibts da was neues?
